worker/scraper.py

```diff
@@ -49,12 +49,12 @@ def scrape_feed_articles(feed):
         # Extract first image from item
         try:
             summary_soup = BeautifulSoup(item.summary)
-            img_url = summary_soup.find('img')['src']
+            img_url = summary_soup.find('img', width_=lambda x: x != 1)['src']
```

Comment:
Ah, very good.

Did you work on the other part the issue: to make sure that the 1px images inside the cards don't stretch out and appear to add "blank" space?